Zombie/ZombieDriver.js

```diff
@@ -1,10 +1,10 @@
 "use strict";
 
-process.on('uncaughtException', e => console.error('Uncaught Exception: ' + e));
-
 const Zombie = require('zombie');
 const zombie = new Zombie();
 
+process.on('uncaughtException', e => console.error('Uncaught Exception: ' + e));
```

Comment:
I think I'd move this right to the bottom of the file - if we ever have issues creating the bridge I'd prefer the process to fail early.